action.yml

```diff
@@ -69,6 +69,9 @@ inputs:
   jwtKeyPassword:
     description: 'Password for key stored in jwtPrivateKey (if needed)'
     required: false
+  jwtGithubAudience:
+    description: ''
```

Comment:
A short description of `jwtGithubAudience` would be helpful. 
For example: _Identifies the recipients that the JWT is intended for_

Reply:
Done. Adjusted description for `jwtGithubAudience` in README as well